src/porepy/utils/porepy_types.py

```diff
@@ -35,3 +37,23 @@
     "pp.fracs.fracture_network_2d.FractureNetwork2d",
     "pp.fracs.fracture_network_3d.FractureNetwork3d",
 ]
+
```

Comment:
I introduced these two types for sake of dynamic creation of compositional variables and phase properties.
Inside `fluid.density()`, the method `phase.density()` is called. While `fluid.density` is a static interface for all other model interfaces, the `phase.density` is created depending on phase context. The static checks and intellisense have now something to work with.